pkg/store/store.go

```diff
@@ -424,16 +424,21 @@ func (s Store) validKnativeIngressClass(objectMeta *metav1.ObjectMeta) bool {
 	return ingressAnnotationValue == s.ingressClass
 }
 
-// ListKnativeIngresses returns the list of TCP Ingresses from
-// configuration.konghq.com group.
+// ListKnativeIngresses returns the list of Knative Ingresses from
+// ingresses.networking.internal.knative.dev group.
 func (s Store) ListKnativeIngresses() ([]*knative.Ingress, error) {
 	var ingresses []*knative.Ingress
 	if s.stores.KnativeIngress == nil {
 		return ingresses, nil
 	}
-	err := cache.ListAll(s.stores.KnativeIngress, labels.NewSelector(),
+
+	err := cache.ListAll(
+		s.stores.KnativeIngress,
+		labels.NewSelector(),
 		func(ob interface{}) {
+			fmt.Printf("\nwithin ListKnativeIngresses cache.ListAll")
 			ing, ok := ob.(*knative.Ingress)
+			fmt.Printf("\nwithin ListKnativeIngresses checking ing %v", ing)
```

Comment:
These look like debug prints that we need to make sure we clean up.

Reply:
my bad. thanks.